### ISSUE TITLE
Can't find /go/src/grpc/*.pb.go

### ISSUE THREAD

**Issue by washanhanzi** [tokopedia/gripmock#65]
I am tring to use gripmock with docker, and mount the `user.proto` file into docker

but the docker exit with:
```
Starting GripMock
Serving stub admin on http://:4771
sed: /go/src/grpc/user.pb.go: No such file or directory
2021/01/23 03:11:35 Fail on sed
```

It seems the pb.go file didn't generated.

**Comment by Kareem-Emad** [comment]
I think the issue happens when you specify the `go_package` option with the full path of your package 
example
```
package service;

option go_package = "grpc/service";
```
will fail with 
```
sed: /go/src/grpc/repository-service.pb.go: No such file or directory
```


removing the option or replacing it with just the package name will work 
```
package service;
option go_package = "service";
```

**Comment by washanhanzi** [comment]
Thank you. It fixes my problem.

**Comment by Kareem-Emad** [comment]
You are welcome

but this still a valid issue since protoc generator complains about the go package missing the fullpath specification, and it will be required in future release so I think that needs to be supported here.

```
WARNING: Deprecated use of 'go_package' option without a full import path in "multi-package/repository-service.proto", please specify:
        option go_package = "multi-package;service";
A future release of protoc-gen-go will require the import path be specified.
See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.
```

**Comment by benquinteros** [comment]
Any update on this?

**Comment by jekiapp** [comment]
Hi @Kareem-Emad 

I'm a bit confused of what is actually being supported in the future. The warning said `WARNING: Deprecated use of 'go_package' option without a full import path` but the one that fixed the issue is specifying package in the short version. I thought full import path means the longer one.

**Comment by jekiapp** [comment]
It turns out a huge incompatibility change. In order to support this, I need to require `go_package` for every proto being used, thus it's non-backward compatible.

**Comment by jekiapp** [comment]
I created this PR #83 to solve this issue, it's quite a big change. It will make a backward incompatibility to the old .proto files since it will require `go_package` declaration.

**Comment by jekiapp** [comment]
I pushed `tkpd/gripmock:v1.10` to address this issue, checkout [this release ](https://github.com/tokopedia/gripmock/releases/tag/v1.10)

**Comment by cchengrbx** [comment]
Hi @jekiapp, thanks for putting out a release.  When I tried v1.10.0, I ended up with a pretty cryptic error. Any idea what the issue might be?

/go/src/grpc/server.go:19:2: package <my_go_package> is not in GOROOT (/usr/local/go/src/<my_go_package>)

I'm executing it like so:
```  my-service:
    container_name: my-service-mock
    image: "docker.artifactory.rbx.com/tkpd/gripmock:${TAG:-v1.10}"
    networks:
      - sendr
    command: "-stub=/stub /proto/my-service-api.proto"
    volumes:
      - >-
        ./my-service-path/proto/:/proto
      - >-
        ./my-service-path/stub/:/stub
    ports:
      - "4770:4770"
      - "4771:4771"```

**Comment by cchengrbx** [comment]
Hi @jekiapp, thanks for pushing the new release to support `go_package`.  After trying it, I get a pretty cryptic error:
```
/go/src/grpc/server.go:19:2: package <my_go_package> is not in GOROOT (/usr/local/go/src/<my_go_package>)
```

Here's how I'm invoking gripmock:
```
# docker-compose.yml
  my-service:
    container_name: my-service-mock
    image: "tkpd/gripmock:${TAG:-v1.10}"
    networks:
      - sendr
    command: "-stub=/stub /proto/my_api.proto"
    volumes:
      - >-
        ./my-service/proto/:/proto
      - >-
        ./my-service/stub/:/stub
    ports:
      - "4770:4770"
      - "4771:4771"
```

Any ideas where I went wrong?

**Comment by jekiapp** [comment]
@cchengrbx did you literally put `<my_go_package>` as value in `go_package` option?
if yes, then you need to change it to a standard path format. Please check this [reference](https://developers.google.com/protocol-buffers/docs/reference/go-generated#package)

**Comment by cchengrbx** [comment]
No, the `go_package` is set to something in the format `foo/barv1` so nothing special.

**Comment by jekiapp** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  I just pushed tkpd/gripmock:1.10.1 try it                                                                                                                                                                                                                                                                                                                        From: ***@***.***: 6 August 2021 23:10To: ***@***.*** to: ***@***.***: ***@***.***; ***@***.***: Re: [tokopedia/gripmock] Unable to use 'go_package' option with full import path in proto file (#65)  
No, the go_package is set to something in the format foo/barv1 so nothing special.

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

Disclaimer:The information contained in this communication is intended solely for the addressed individual or entity only. If you are not the intended recipient, you are hereby notified that any disclosure, copying, distribution  or taking any action in reliance on the contents of this information is strictly prohibited and may be unlawful.If you have received this communication in error, please notify us immediately by responding to this email or contact us at the address mentioned above.

**Comment by cchengrbx** [comment]
Thanks @jekiapp, `.1.10.1` is working great.
